Dockerfile

```
@@ -1,27 +1,23 @@
-# Application build stage
-FROM golang:1.21-alpine3.18 as build
-
-ENV GOPATH /go
-
-COPY . /go/src/github.com/equinor/radix-networkpolicy-canary/
+FROM golang:1.21-alpine3.18 as builder
```

```suggestion
FROM golang:1.21-alpine3.19 as builder
```
3.19 is latest version of alpine. Should we upgrade to this one instead?